From 763625ce265dca55a1b66fc9142c7523f7062fc3 Mon Sep 17 00:00:00 2001
From: Voon Weifeng <weifeng.voon@intel.com>
Date: Sat, 27 Jul 2019 07:53:45 +0800
Subject: [PATCH 3/4] net: stmmac: add EHL SGMII 2.5Gbps PCI info and PCI ID

Add EHL SGMII 2.5Gbps PCI info and PCI ID
Port from Yocto mainline:
https://github.com/intel/linux-intel-lts/commit/c31f9866d6763ce9abdf4b077de2104980a820fb

Tracked-On: OAM-99178
Signed-off-by: Voon Weifeng <weifeng.voon@intel.com>
Signed-off-by: Ong Boon Leong <boon.leong.ong@intel.com>
Change-Id: Ib4631ead6b2b7cdcf5ecafd3ea9ffd747f6a9fb1
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
index 81330453fd0a..86636edc8305 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
@@ -646,6 +646,7 @@ static SIMPLE_DEV_PM_OPS(stmmac_pm_ops, stmmac_pci_suspend, stmmac_pci_resume);
 #define STMMAC_DEVICE_ID 0x1108
 #define STMMAC_EHL_RGMII1G_ID	0x4b30
 #define STMMAC_EHL_SGMII1G_ID	0x4b31
+#define STMMAC_EHL_SGMII2G5_ID	0x4b32
 #define STMMAC_EHL_PSE0_RGMII1G_ID	0x4ba0
 #define STMMAC_EHL_PSE0_SGMII1G_ID	0x4ba1
 #define STMMAC_EHL_PSE0_SGMII2G5_ID	0x4ba2
@@ -666,6 +667,7 @@ static const struct pci_device_id stmmac_id_table[] = {
 	STMMAC_DEVICE(INTEL, STMMAC_QUARK_ID, quark_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_RGMII1G_ID, ehl_rgmii1g_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_SGMII1G_ID, ehl_sgmii1g_pci_info),
+	STMMAC_DEVICE(INTEL, STMMAC_EHL_SGMII2G5_ID, ehl_sgmii1g_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_PSE0_RGMII1G_ID,
 		      ehl_pse0_rgmii1g_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_PSE0_SGMII1G_ID,
--
2.17.1

